packages/beacon-state-transition/src/fast/util/cachedBalanceList.ts

```diff
@@ -34,7 +34,7 @@ export class CachedBalanceList implements List<Gwei> {
   }
 
   updateAll(balances: BigUint64Array): void {
-    this.persistent.vector = Vector.from(balances);
+    this.persistent.vector = PersistentVector.from(balances);
```

Comment:
I suppose we call this from epoch transition only. Can we use the new `TransientVector.from()` instead?

Reply:
`PersistentVector.from` uses `TransientVector` behind the scenes :)
https://github.com/ChainSafe/persistent-ts/blob/master/src/Vector.ts#L72